bitchat/Services/BluetoothMeshService.swift

```diff
@@ -2232,13 +2232,40 @@ class BluetoothMeshService: NSObject {
 extension BluetoothMeshService: CBCentralManagerDelegate {
     func centralManagerDidUpdateState(_ central: CBCentralManager) {
         // Central manager state updated
-        if central.state == .poweredOn {
+        switch central.state {
+        case .poweredOn:
             startScanning()
             
             // Send announces when central manager is ready
             DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) { [weak self] in
                 self?.sendBroadcastAnnounce()
             }
+        case .poweredOff:
+            // Bluetooth is turned off, notify delegate
+            if let vm = delegate as? ChatViewModel {
+                DispatchQueue.main.async {
+                    vm.isConnected = false
+                    vm.connectedPeers.removeAll()
+                    
+                    // Add system message about Bluetooth being off
+                    let systemMessage = BitchatMessage(
+                        sender: "system",
+                        content: "bluetooth is turned off. please enable bluetooth to connect with other users.",
```

Comment:
content: "Bluetooth is turned off. Please enable Bluetooth to connect with other users."